src/telegrafs/components/CollectorCard.tsx

```diff
@@ -76,8 +77,9 @@ class CollectorRow extends PureComponent<
         />
         <ResourceCard.Meta>
           <Link
-            to={`/orgs/${org.id}/load-data/telegrafs/${collector.id}/instructions`}
+            to=""
```

Comment:
Does this do anything unintended or is it harmless to have a blank string for the `to` prop?

Reply:
I looked into this and didn't find anything that discourages passing an empty string to the `to` prop. The overlay opens as expected when the link is clicked with no errors logged in the console. An alternative is to use a button and style it as a link.

Reply:
@blegesse-git I think the proper move is to remove this `Link` component and replace it with something that doesn't have an `href` (`<Link to="http://example.com">` gets converted to something like `<a href="http://example.com">`), maybe like you suggested with a button. Previously, the behavior of this `Link` relied on

```ts
<Route
  path={`${telegrafsPath}/:id/instructions`}
  component={TelegrafInstructionsOverlay}
/>
```

existing, and rendering an overlay when the url matched. Since this PR triggers the overlay via redux, I think leaving this `Link` might have strange behavior

Reply:
okay no problem! It's probably better to take the safe approach. I'll replace it with a button and style it accordingly.